README.md

```diff
@@ -83,54 +83,77 @@ invariants.
 	ASSERT_NE(MAIN, left, right, "left must not equal right");
 	ASSERT_TRUE(MAIN, param, "param must be true");
 	ASSERT_FALSE(MAIN, param, "param must be false");
-## Build logger
-To build logger run the following command
-1. `make`
-2. Verify the test-bin is created
-3. Run the following command
-   1. g++ test.cc -o test.exe
 
 
-# Jenkins Docker image setup
-Reference https://www.jenkins.io/doc/book/installing/docker/ for more information
+# Automated Testing Environment
+The testing environment will use jenkins running in docker to deploy containers with logger running
+The following instructions will guide the user/tester to build and deploy the application within a local environment or a container.
 
-## Prerequisites
-- Linux (centos prefered) OS VM
-- docker
+# Prerequisite
 
-## Install Jenkins Docker container
-The following will install a Jenkins docker container with persistent volume allowing for saved states based on the container being brought up and down.
+## Prerequisites Windows
 
-It is recommended to deploy the jenkins docker container within a Linux VM. For this deployment we will deploy jenkins within a docker container within a Centos VM.
+The following needs to be pre installed on a Windows 10 workstation  before proceeding. How to install the services are outside of the scope of this README.
 
+- Virtualbox (hypervisor)- See Virtualbox Prerequisite
+- - Linux (CentOS prefered) OS VM image
 
-### Prerequisite
+### Virtualbox Prerequisite
+The following need to be installed on the Linux OS VM that is installed on virtualbox. The installation of how to install the OS is outside of the scope of this readme.
+- doxygen - for building logger
+- Install g++ and gcc
+- Install Make
 - Create a docker login at https://hub.docker.com
+- **sshkeygen**- for ssh connection
+- **docker**- for building containers
+
+
+## Jenkins Docker image setup
+Reference https://www.jenkins.io/doc/book/installing/docker/ for more information
 
-###installation
+## Install Jenkins Docker container
+The following will install a Jenkins docker container with persistent volume allowing for saved states based on the container being brought up and down.
+
+It is recommended to deploy the jenkins docker container within a Linux VM. For this deployment we will deploy jenkins within a docker container within a Centos VM.
```

Comment:
Maybe combine these sentences to be more clear - maybe the first sentence isn't needed.  Is the first sentence saying the Jenkins Docker container within a Linux VM. Then the second sentence says we used the Centos flavor of Linux?

Reply:
Overall comment - capitalize Jenkins